aten/src/ATen/function_wrapper.py

```diff
@@ -492,6 +500,7 @@ def __getitem__(self, x):
     'return': ReturnDecl,
     'returns': List[ReturnType],
     'scalar_check': str,
+    'schema': str,
```

Comment:
can you work out with @cpuhrsch how to connect this to what he's doing?  This is fine to get in for now (with a comment) but we need to unify these things.